sros/docker/launch.py

```diff
@@ -35,23 +34,47 @@ def trace(self, message, *args, **kws):
 
 logging.Logger.trace = trace
 
+# Default power type to use, SROS complains
+# "Class Power Shelf : wrong type inserted" when using "ac/hv" here
+DEFAULT_POWER = "dc"
+
+
+def LINE_CARD(chassis, card, mda, integrated=False, card_type=None):
+    """
+    Configures a line card in a distributed or integrated deployment model.
+    Note that it may be possible to omit configuring these explicitly
+    """
+    slot = "A" if integrated else "1"
+    return {
+        "timos_line": f"slot={slot} chassis={chassis} card={card} mda/1={mda}",
+        "card_config": f"""
+       /configure card 1 card-type {card_type if card_type else card}
+       /configure card 1 mda 1 mda-type {mda}
+      """,
+    }
```

Comment:
I am doing some comments to the parts that have been surgically merged in other PRs